source/industry/Fanuc.rst

```diff
@@ -0,0 +1,121 @@
+Fanuc
+=====
+
+.. _manual_collection: http://cncmanual.com/fanuc-robotics/
+.. _roboguide_help: http://cncmanual.com/download/4472/
+.. _reference_manual: http://cncmanual.com/fanuc-robotics-r-30ia-controller-karel-reference-manual/
+.. _manual_slides: http://www.lakos.fs.uni-lj.si/wp-content/uploads/2017/12/Fanuc-robot.pdf
+.. _Karel: https://www.tristarcnc.com/News/KarelProgrammingLanguage
+
+Cartesian trajectories for Fanuc robots
+
+  =================================   =======================================
+  Teach pendant                       FANUC iPendant touch
+  Programming / simulation software   ROBOGUIDE
+  Software                            
+  User interface                      Fanuc iHMI (Intelligent Human Machine Interface)
+  Programming language                FANUC `Karel`_ (derived from Pascal)
```

Comment:
Karel is a programming language on Fanucs, but it's not used to control any motion.

Much of the content in this page seems to focus on system variables which do influence motion, but are typically not directly changed by user code.

TP is the programming language which is used for motion programs. It almost never directly changes the system variables mentioned in the *Specification of velocity* section (and further down).

Reply:
@stefanscherzinger @LeaStef Would one of you please work that in?

Reply:
```suggestion
  Programming language                TP (Teach Pendant), Karel
```

@LeaStef I put your suggestion here to keep it in this discussion thread.

@gavanderhoorn Would hit your point raised? I'm not sure whether you are also stating that the content below might not reflect the relevant aspects.

Reply:
The system variables mentioned in this article are "never" directly manipulated from/in TP programs.

They encode system configuration, which is manipulated through menus, keys on the TP and/or statements in a TP program, but never directly (ie: you don't really write `$MCR[].$GENOVERRIDE = 100` in TP, but `OVERRIDE=100%`).

I'm not saying those system variables must not be mentioned, I'm asking (implicitly) why they *are* mentioned. As, as I wrote, user programs don't use them to configure motion. Nor would any driver I believe (as motion *must* be initiated/controlled from TP, and so drivers would also use TP for that).

Reply:
Sorry, for taking s long. After all I agree and adapted it accordingly.